data/tests.yml

```diff
@@ -0,0 +1,570 @@
+- way_id: 159276650
```

Comment:
The advantages of YAML seem to be
1) less "noise" for quickly writing things like output in a text editor
2) Proper comments

Were there other motivations?

Reply:
+ easier to write by hand
-  yaml is really bad (ideally i would use https://hitchdev.com/strictyaml/why/, but support is less good)